packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -27,15 +27,23 @@ import { Translation } from './Translation';
 
 const SPACING = 6;
 
-const Container = styled.div`
+const Container = styled.div<{ isClickable: boolean }>`
     display: flex;
+    justify-content: space-between;
```

Comment:
this is not needed, everything spreados out when `View` is not rendered because of this 🙏